signalfx-org-metrics/metrics.yaml

```diff
@@ -1,3 +1,273 @@
+sf.org.apm.numSpansDroppedThrottle:
+  brief: The number of spans SignalFx accepts beyond the allowed ingest volume.
+  description: 'The number of spans SignalFx accepts beyond the allowed ingest volume.
+    SignalFx drops spans it receives after the ingestion volume limit is reached.
+
+    Dimension(s): `orgId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedThrottle
+
+sf.org.apm.numSpansDroppedThrottleByToken:
+  brief: The number of spans SignalFx accepts for a specific access token
+    beyond the allowed ingest volume.
+  description: 'The number of spans SignalFx accepts for a specific access token
+    beyond the allowed ingest volume. SignalFx drops spans it receives after
+    the ingestion volume limit is reached.
+
+    Dimension(s): `orgId`, `tokenId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedThrottleByToken
+
+sf.org.apm.grossSpansReceived:
+  brief: The number of spans SignalFx receives before filtering and throttling.
+  description: 'The number of spans SignalFx receives before filtering or throttling.
+
+  Dimension(s): `orgId`
+
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpansReceived
+
+sf.org.apm.grossSpansReceivedByToken:
+  brief: The number of spans SignalFx receives for a specific access token
+    before filtering and throttling.
+  description: 'The number of spans SignalFx receives for a specific access token
+    before filtering or throttling.
+
+    Dimension(s): `orgId`, `tokenId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpansReceivedByToken
+
+sf.org.apm.numSpansReceived:
+  brief: The number of spans SignalFx accepts after filtering and throttling.
+  description: 'The number of spans SignalFx accepts after filtering and throttling.
+
+  Dimension(s): `orgId`
+    
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansReceived
+
+sf.org.apm.numSpansReceivedByToken:
+  brief: The number of spans SignalFx accepts for a specific access token
+    after filtering and throttling.
+  description: 'The number of spans SignalFx accepts for a specific access token
+    after filtering and throttling.
+
+  Dimension(s): `orgId`, `tokenId`
+    
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansReceivedByToken
+
+sf.org.apm.invalidSpansReceived:
+  brief: 
```

Comment:
I believe this metric the one right below it is currently being generated in TraceIngest. @mdubbyap could you please help define "invalid" here?

Also, as a side note, after traceIngest we do have a couple places we do have a couple of checks that reject trace for analysis or storage purposes. Those are not currently included in this metric. It might be beneficial if we could start a doc to discuss what those are and how can they affect the metrics we report to the customer.

Reply:
It is not generated in trace-ingest. Which is why i pointed to the backend. Trace ingest emits sf.org.apm.numSpansDroppedInvalid

Reply:
This metric is not generated in the backend though. I cannot find any reference to it in the backend? Maybe PG @mpetazzoni or maybe drop?